earth_enterprise/rpms/build.gradle

```diff
@@ -0,0 +1,138 @@
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "com.netflix.nebula:gradle-ospackage-plugin:4.4.2"
+    }
+}
+
+apply plugin: "nebula.ospackage"
+
+def env = System.getenv()
+def home = env['HOME']
+def tmp_install = '/tmp/fusion_os_install'
+def tmp_common = tmp_install + '/common'
+def tmp_common_usermagic = tmp_install + '/common/user_magic'
+def tmp_manual = tmp_install + '/manual'
+def tmp_server_opt = tmp_install + '/server/opt'
+def tmp_server_etc = tmp_install + '/server/etc'
+def tmp_server_apachesupport = tmp_install + '/server/AppacheSupport/'
+def tmp_server_usermagic = tmp_install + '/server/user_magic/'
+
+def tmp_fusion = tmp_install + '/fusion'
+
+// def tmp_searchexample = tmp_install + 'searchexample'
+//geplaces
+
+def base_install_dir = "/test"
+
+apply plugin: 'nebula.ospackage'
+
+ospackage {
+    packageName = 'ge-common'
+    version = '5.2.1'
+    release = '1'
+    arch = X86_64
+    os = LINUX
+    into '/opt/google'
+}
+
+task commonRpm (type: Rpm) {
+    packageName = 'ge-common'
+
+    from(tmp_common) {
```

Comment:
We need to split `postgis` off from the rest of the third-party libraries, because of its license, but we don't have address that on this ticket. And that would go away when we unbundle PostGRE and PostGIS.